js/index.js

```diff
@@ -42,7 +50,77 @@ let currentPosition = 0;
 const carrouselArrows = document.querySelectorAll(".carrousel__arrow");
 /* Take actual image from document */
 const carrouselImage = document.querySelector(".carrousel__img");
+
+ function automatic() {
```

Comment:
This function could have a more indicative name like `automaticCarrousel` or `automaticChange`